### DISCUSSION TITLE
Get Selected value?

### DISCUSSION THREAD

**Discussion by abdelilahdourhmani** [kevlatus/flutter_fortune_wheel#105] [Q&A]
Hi dear,

How can i get the selected value ?

**Comment by adrian-martos** [comment]
Hi @abdelilahdourhmani i solved it.

I`ll show you how:

@override
  Widget build(BuildContext context) {

    int value = Fortune.randomInt(0, widget.teams.length);
    List<FortuneItem> fortuneItems = widget.teams.map((item) {
      return FortuneItem(child: Text(item['name']));
    }).toList();

    return Scaffold(
      appBar: AppBar(
        title: const Text('Selección de equipo'),
      ),
      body: GestureDetector(
        child: Column(
          children: [
            Expanded(
                child: FortuneWheel(
              onAnimationEnd: () => { 
                print(" ${widget.teams[value]['name']}"); 
              },
              duration: const Duration(seconds: 3),
              animateFirst: false,
              selected: Stream.value(value),
              items: fortuneItems,
            ))
          ],
        ),
      ),
    );
  }
